internal/configs/removed.go

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package configs
+
+import (
+	"github.com/hashicorp/hcl/v2"
+	"github.com/opentofu/opentofu/internal/addrs"
+)
+
+type Removed struct {
+	From *addrs.RemoveEndpoint
```

Comment:
The variable name `From` feels strange here, maybe because we're mimicing some of the `move` logic.

Maybe the noun to use here as the target of removal is `Target` or `Address` or `Resource`?

Reply:
The `From` variable name here references the `From` attribute of the removed block:
```
removed {
  from = module.example.null_resource.example_resource
}
```

It's using the same convention as in `moved.go` and `import.go`. So, IMO it makes sense, but let me know if you think otherwise.